imports/plugins/core/ui-navbar/client/components/navbar/navbar.js

```diff
@@ -90,27 +92,36 @@ Template.CoreNavigationBar.helpers({
     };
   },
 
-  tagNavProps() {
+  tagNav() {
```

Comment:
This function should be moved into the Tag Nav container in a compose function. Which would pass data into the container as props. 
Example code: https://github.com/reactioncommerce/reaction/blob/brent-milestone-54/client/modules/i18n/templates/currency/containers/currencyContainer.js#L44